### DISCUSSION TITLE
Gopro 10 hangs on the third record

### DISCUSSION THREAD

**Discussion by Sekilsgs2** [gopro/labs#256] [General]
Hi..

I'm buy used GP10 with broken main lcd, but front lcd working normal.
I'm install 1.50 labs firmware.
I'm make full reset via qrcodes and make full reset via menu (main lcd touch screen working - according to the instructions, just clicked where necessary menu).
After reset i'm connect camera to app via qr code !PA.
And it turned out that after recording two videos and clicking on the recording of the third - the camera hangs - the application stops showing the video, does not respond to the buttons, but on the front screen, the video continues to be shown in real time. It helps either to hold the button for 1 second or to pull out the battery.
There is no such thing on the stock firmware - help - what could it be?

**Comment by dnewman-gpsw** [comment]
Only with set DLRV set to one.  As there is no longer any performance benefits to using DLRV (there was a bit in older firmware), so you really shouldn't use this command.  However is it for having a cleaner and more structured SD card, then use PRXY=1 instead.
